src/net.cpp

```diff
@@ -1502,6 +1503,57 @@ void CConnman::ThreadAddressSeed()
     else{
         LogPrintf("DNS seeding disabled\n");
     }
+    // Fixed seeds
+    bool add_fixed_seeds = gArgs.GetBoolArg("-fixedseeds", DEFAULT_FIXEDSEEDS);
+
+    if (!add_fixed_seeds) {
+        LogPrintf("Fixed seeds are disabled\n");
+        return;
+    }
+    while ( addrman.size() == 0) {
```

Comment:
nit: remove space before `addrman`

Reply:
(Friendly tip, one can avoid nitty formatting issues by running clang format on commits before pushing, see `/contrib/devtools/clang-format-diff.py` for details.)

Reply:
Done, thanks!